.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Tests
+on: [ push, pull_request ]
+jobs:
+  test:
+    name: Test (Ruby ${{ matrix.ruby }}, Rails ${{ matrix.rails }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.0', '2.1', '2.2', '2.3', '2.4', '2.5', '2.6', 'jruby-9.1' ]
+        rails: [ '4.0', '4.1', '4.2', '5.0', '5.1', '5.2' ]
+        exclude:
+          - { ruby: '2.0', rails: '5.2' }
+          - { ruby: '2.0', rails: '5.1' }
+          - { ruby: '2.0', rails: '5.0' }
+          - { ruby: '2.1', rails: '5.2' }
+          - { ruby: '2.1', rails: '5.1' }
+          - { ruby: '2.1', rails: '5.0' }
+    env:
+      BUNDLE_GEMFILE: gemfiles/rails-${{ matrix.rails }}.gemfile
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - run: bundle exec rake
```

Comment:
Should it not have rspec in run command?

Reply:
In the [Rakefile](https://github.com/sdtechdev/flip/blob/e965bdedde202a921c3616aebdc3535db375a15d/Rakefile#L5), there is a `default` task that fallback to the `spec` one
```
task :default => :spec
```
And the `spec` task runs all specs.

It works fine, as you can see in [my forked repository](https://github.com/MassimilianoLattanzio/flip/actions).

So maybe it could depend on some missing permissions?